zxcvbn-js.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = Zxcvbnjs::VERSION
 
-  gem.add_dependency 'mini_racer'
+  gem.add_dependency 'mini_racer', '~> 0.5'
```

Comment:
So -- to clarify -- this isn't strictly necessary in order to get consuming apps on a newer version of `mini_racer`, because they are free to `bundle update mini_racer` on their own and whatever is in their lockfiles is the version they'll ultimately use, right? Or am I missing something.

Reply:
My hypothesis where we both tried to bump mini_racer and had build issues was that by bumping mini_racer here we could remove it from the Gemfiles there and possibly sidestep those build issues. AFAICT, it only exists as a transitive dependency of this gem.

Reply:
Ah interesting. I had thought we might also be relying on it for javascript asset pipeline compilation use-cases.

Reply:
I'm pretty sure that's the case actually, so it's not _just_ a transitive dependency for us, I believe.

Reply:
ah, good catch, @samandmoore. In that case, I think this is probably a healthy but not fully ameliorative change for the other use case. 😞